### DISCUSSION TITLE
git clones - We don’t have enough data to show anything useful.

### DISCUSSION THREAD

**Discussion by ltspicer** [community/community#38358] [Repositories] [answered]
Hi friends

Why can't I see "cloners" statistics?

![Auswahl_005](https://user-images.githubusercontent.com/11616336/200125254-07e8ab28-8823-45a3-ad03-7b9beca423ce.png)

Link to the project:
https://github.com/ltspicer/dxf2gcode.linux.installer

**Comment by radar-blip** [comment]
Probably because there's no cloning going on. 
It's like the message says, it takes time to populate this graph.

**Comment by ltspicer** [comment]
I do not understand that. The content has been online for a long time and has many downloads.

**Comment by airtower-luna** [comment, marked as answer]
Do people actually clone the repository? If they only download the script file it won't show up in the clone statistics.

**Comment by ltspicer** [comment]
Is there a way to see the number of downloads?
